src/main/scala/ai/eto/rikai/sql/parser/RikaiExtAstBuilder.scala

```diff
@@ -42,6 +42,16 @@ class RikaiExtAstBuilder extends RikaiSqlBaseBaseVisitor[AnyRef] {
       }
     }
 
+  protected def parseOptionList(ctx: OptionListContext): Map[String, String] = {
+    if (ctx != null) {
```

Comment:
can ctx be null?

Reply:
Yes, here we use `OptionListContext`, if create model query doesn't have `OPTIONS`, this will be a null.